src/modules/avformat/filter_avfilter.c

```diff
@@ -58,6 +58,11 @@ typedef struct
 	int reset;
 } private_data;
 
+static inline int non_animated(const char *value)
```

Comment:
I wonder if we should consider adding this to mlt_properties. If we do that, would it be possible to check the property for an animation object instead of parsing a string?

Reply:
When the life a property begins as a string such as from XML or `melt`, there is no animation object until the property has been accessed using an anim getter, which causes it to be parse into `mlt_animation`. However, yes, we can consider moving this into a common property.

Reply:
I noticed a number of functions in `mlt_property.c` are applying a similar heuristic. Thus, it totally makes sense to refactor this into a common function, and I will.

Reply:
done

Reply:
done